tests/translator/test_translator.py

```diff
@@ -298,7 +298,6 @@ class TestTranslatorEndToEnd(TestCase):
                 "state_machine_with_api_resource_policy",
                 "state_machine_with_api_auth_default_scopes",
                 "state_machine_with_condition_and_events",
-                "state_machine_with_xray",
```

Comment:
Why this line is deleted?

Reply:
Seems like a merge from master. Reverted